apps/zbugs/api/index.ts

```diff
@@ -220,8 +217,8 @@ async function getQueriesHandler(
     reply.send(
       await handleTransformRequest(
         (name: string, args: ReadonlyJSONValue | undefined) => {
-          const query = queryRegistry.mustGet(name, authData);
-          return query(args);
+          const query = mustGetQuery(queries, name);
+          return query(args).toQuery(authData);
```

Comment:
Weird for a thing called `query` to have a method called `toQuery`. Should be `toZQL`.

Reply:
I had `create` at first but that didn't seem right either.


There is precedence for having A.toA().

`String.prototype.toString`
`Iterateror.prototype[Symbol.iterator]`


But lets iterate on the name.

- toZQL(ctx)
- zql(ctx)
- attach(ctx)
- context(ctx)
- withContext(ctx)
- create(ctx)
- make(ctx)